src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -44,474 +62,326 @@ public class AdjournCaseIt extends AbstractEventIt {
     @MockBean
     private UserDetails userDetails;
 
+    @DisplayName("Call to mid event callback when path is YES NO YES will validate the data when due date in past")
     @Test
-    public void callToMidEventCallback_whenPathIsYesNoYes_willValidateTheData_WhenDueDateInPast() throws Exception {
+    public void testMidEventCallbackWithDueDateInPast() throws Exception {
         setup();
-        setJsonAndReplace("callback/adjournCaseGeneratedVideoWhenCaseNotListedStraightAwayWithDirectionsMade.json", Arrays.asList("DIRECTIONS_DUE_DATE_PLACEHOLDER"),
-            Arrays.asList("2019-10-10"));
+        setJsonAndReplace(NOT_LISTED_STRAIGHT_AWAY_WITH_DIRECTIONS_MADE_JSON,
+            List.of(DIRECTIONS_DUE_DATE_PLACEHOLDER),
+            List.of(DATE_2019));
 
         MockHttpServletResponse response = getResponse(getRequestWithAuthHeader(json, "/ccdMidEvent"));
         assertHttpStatus(response, HttpStatus.OK);
         PreSubmitCallbackResponse<SscsCaseData> result = deserialize(response.getContentAsString());
 
-        assertEquals(1, result.getErrors().size());
-        assertEquals("Directions due date must be in the future", result.getErrors().toArray()[0]);
+        assertThat(result.getErrors()).hasSize(1);
+        assertThat(result.getErrors()).containsOnly("Directions due date must be in the future");
     }
 
+    @DisplayName("Call to mid event callback when path is YES NO YES will validate the data when due date in future")
     @Test
-    public void callToMidEventCallback_whenPathIsYesNoYes_willValidateTheData_WhenDueDateInFuture() throws Exception {
+    public void testMidEventCallbackWithDueDateInFuture() throws Exception {
```

Comment:
See comment on PreviewDocumentServiceTest re: test method naming and apply to rest of class.

Reply:
ok